client/ayon_core/plugins/load/delivery.py

```diff
@@ -282,10 +290,10 @@ def _get_templates(self, anatomy):
         """Adds list of delivery templates from Anatomy to dropdown."""
         templates = {}
         for template_name, value in anatomy.templates["delivery"].items():
-            path_template = value["path"]
+            directory_template = value["directory"]
             if (
-                not isinstance(path_template, str)
-                or not path_template.startswith('{root')
+                not isinstance(directory_template, str)
```

Comment:
I guess the `str` type check is not necessary anymore.

Reply:
Should be resolved.